server/eclipse-project/src/main/java/us/freeandfair/corla/model/CountyDashboard.java

```diff
@@ -515,6 +515,14 @@ public Round currentRound() {
     }
   }
 
+  /** all the cvr ids that have been selected from previous rounds **/
+  public Set<Long> previouslyAudited() {
```

Comment:
Depending on when this gets called, it may not be true, because if this gets called when a round is in progress it will count ballots from that round. I think this has to check the round status somehow, even though I know you wrote it with a single use case in mind.

Reply:
good point!